test/lish/many_files_test.dart

```diff
@@ -35,7 +33,7 @@ void main() {
       'the command line', () async {
     await d.validPackage.create();
 
-    int argMax;
+    num argMax;
     if (Platform.isWindows) {
       // On Windows, the maximum argument list length is 8^5 bytes.
       argMax = math.pow(8, 5);
```

Comment:
Instead of using `num`, maybe it's better to do 
```suggestion
      argMax = math.pow(8, 5).toInt();
```
Or even just:
```suggestion
      argMax = 32768; // 8^5
```

Reply:
Done